.github/workflows/godwoken-tests.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches: [develop, master, ci, 'v1*']
     tags: ["v*.*.*"]
+    paths:
+      - 'gwos-evm/**'
```

Comment:
Doesn't this mean that this test is only triggered when `gwos-evm` changes? Doesn't feel right.

Reply:
You are right, the paths should be removed

Reply:
Fixed in https://github.com/godwokenrises/godwoken/pull/950/files